tests/unit/serve/dynamic_batching/test_dynamic_batching_config.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from jina import Document, DocumentArray, Executor, Flow, dynamic_batching, requests
+
+
+class MyExecutor(Executor):
+    @requests(on=['/cat', '/kitten'])
+    @dynamic_batching(preferred_batch_size=4, timeout=2000)
+    def cat_fun(self, docs, **kwargs):
+        return DocumentArray([Document(text='cat')])
+
+    @requests()
+    @dynamic_batching(preferred_batch_size=10, timeout=10000)
+    def default_fun(self, docs, **kwargs):
+        return DocumentArray([Document(text='bar')])
+
+
+def test_dynamic_batching_config(tmpdir):
+    TMPDIR: Path = Path(tmpdir)
```

Comment:
this is not a unit test, this is an integration test, and it should be reacher. Does it work when no `dynamic_batching` decorator exist?

Reply:
This is not a unit test move to integration